Source/Atomic/UI/UIEditField.h

```diff
@@ -72,6 +72,10 @@ class UIEditField : public UIWidget
 
     virtual bool OnEvent(const tb::TBWidgetEvent &ev);
 
+    virtual void OnFocusChanged(bool focused);
+
+    // Used to keep track of if we have just been focused for the click select
+    bool firstFocusFlag_;
```

Comment:
C/C++ are low level and variables must be explicitly initialized otherwise they are whatever happens to be in memory, here is an example of doing so in the constructor (outside of the body which is preferred): https://github.com/AtomicGameEngine/AtomicGameEngine/blob/b3267990cc6724d60e5333f00e795e46533feede/Source/Atomic/UI/UI.cpp#L117